--- ./filezilla/data/Makefile.am	2021-10-06 12:45:14.507285600 -0500
+++ ./filezilla/data/Makefile.am-patched	2021-10-06 12:46:52.352386600 -0500
@@ -25,11 +25,11 @@
 	@rm -f dll_gui_uninstall.nsh
 	@rm -rf dlls_gui
 	@mkdir -p dlls_gui
-	"$(srcdir)/dllcopy.sh" gui "$(top_builddir)/src/interface" filezilla.exe "$(OBJDUMP)" "$(CXX)" "../src/engine/:../src/commonui/:$$PATH" "$(WX_LIBS)"
-	"$(srcdir)/dllcopy.sh" gui "$(top_builddir)/src/putty" fzsftp.exe "$(OBJDUMP)" "$(CXX)" $$PATH
-	"$(srcdir)/dllcopy.sh" gui "$(top_builddir)/src/putty" fzputtygen.exe "$(OBJDUMP)" "$(CXX)" $$PATH
+	"$(srcdir)/dllcopy.sh" gui "$(top_builddir)/src/interface" filezilla.exe "$(OBJDUMP)" "$(CXX)" "../src/engine/:../src/commonui/:$(prefix):$(bindir):$(libdir)" "$(WX_LIBS)"
+	"$(srcdir)/dllcopy.sh" gui "$(top_builddir)/src/putty" fzsftp.exe "$(OBJDUMP)" "$(CXX)" $(prefix):$(bindir):$(libdir)
+	"$(srcdir)/dllcopy.sh" gui "$(top_builddir)/src/putty" fzputtygen.exe "$(OBJDUMP)" "$(CXX)" $(prefix):$(bindir):$(libdir)
 	if test -f "${top_builddir}/src/storj/fzstorj.exe"; then \
-		"$(srcdir)/dllcopy.sh" gui "$(top_builddir)/src/storj" fzstorj.exe "$(OBJDUMP)" "$(CXX)" $$PATH; \
+		"$(srcdir)/dllcopy.sh" gui "$(top_builddir)/src/storj" fzstorj.exe "$(OBJDUMP)" "$(CXX)" $(prefix):$(bindir):$(libdir); \
 	fi
 
 installlangfiles.nsh: Makefile

